test/integration/site/_collections/index.js

```diff
@@ -0,0 +1,22 @@
+const fs = require('fs');
+const path = require('path');
+const runEleventy = require('./runEleventy');
+
+describe.only('_collections', function() {
+  describe('in DEV env', function() {
+    before(async function() {
+      // Disable the timeout as it'll take build a little while to finish.
+      // eslint-disable-next-line
+      this.timeout(0);
+
+      await runEleventy('dev');
```

Comment:
Please note that this is implemented in https://github.com/GoogleChrome/web.dev/pull/2640